<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPointDialog</class>
 <widget class="QDialog" name="NewPointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="edit_x">
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="edit_y">
     <property name="minimum">
      <double>-1000000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
